### PR TITLE
publiccloud: Measure boot time of PC VM

### PR DESCRIPTION
**Pull request by asmorodskyi**
publiccloud : Measure boot time of PC VM

### CODE REVIEW

**Comment by okurz** [comment]
Please delete the space before `:` in the commit message and PR description

**Comment by asmorodskyi** [comment]
this commit is depending on #6458 and should be merged only after it , so putting this one on pause before #6458 will get merged

File: tests/publiccloud/ipa.pm [hunk 1]
```diff
@@ -29,6 +29,14 @@ sub run {
         results_dir => 'ipa_results'
     );
 
+    if (get_var('PUBLIC_CLOUD_CHECK_BOOT_TIME')) {
+        my $kernel_max_boot_time = 60;
+        my $system_max_boot_time = 120;
+        my $out                  = script_output('grep "^Startup finished in" ' . $ipa->{logfile});
+        die 'Fail to find boot time in log' unless $out =~ /Startup finished in (\d{1,4}\.\d{3})s \(kernel\) \+ \d{1,4}\.\d{3}s \(initrd\) \+ \d{1,4}\.\d{3}s \(userspace\) = (\d{1,4}\.\d{3})s/;
+        die 'Kernel boot is too slow - ' . $1          if $1 > $kernel_max_boot_time;
+        die 'Overall system boot is too slow  - ' . $2 if $2 > $system_max_boot_time;
```
Comment (jlausuch) [inline tests/publiccloud/ipa.pm]: would be nice to use record_info() about booting time for openqa job.

Comment (cfconrad) [inline tests/publiccloud/ipa.pm]: For me it is ok to use the log file from ipa, to get that information's.
We should add a extra threshold for wicked startup time of 30s, so we catch wicked timeouts.

Comment (asmorodskyi) [reply tests/publiccloud/ipa.pm]: Done

**Comment by asmorodskyi** [comment]
prove runs for latest version of code http://kimball.arch.suse.de/tests/385 
http://kimball.arch.suse.de/tests/384

**Comment by jlausuch** [comment]
LGTM

**Comment by asmorodskyi** [comment]
prove run for latest code version http://kimball.arch.suse.de/tests/388#step/ipa/113

**Pull request merged**